### PR TITLE
[14.0][MIG] stock_account_quantity_history_location

### PR DESCRIPTION
**Pull request by atchuthan**
minor migration depends on https://github.com/OCA/stock-logistics-reporting/pull/156

@SodexisTeam

### CODE REVIEW

**Review by ypapouin — COMMENTED** [review]
LGTM - Code Review
This PR depends on https://github.com/OCA/stock-logistics-reporting/pull/156, as mentioned, to pass Travis and Runbot CI, but there is not functional changes since 13.0 so it should be Ok.

**Comment by atchuthan** [comment]
@ypapouin https://github.com/OCA/stock-logistics-reporting/pull/156 merged.  This MR is ready for review.

File: stock_account_quantity_history_location/readme/CONSTRIBUTORS.rst [hunk 1]
```diff
@@ -3,3 +3,5 @@
 * `Tecnativa <https://www.tecnativa.com>`_:
 
   * Víctor Martínez
+
+* Sodexis <dev@sodexis.com>
```
Comment (dreispt) [inline stock_account_quantity_history_location/readme/CONSTRIBUTORS.rst]: I noticed the filename has a type. Can you rename it to CONTRIBUTORS.rst please?

Comment (atchuthan) [reply stock_account_quantity_history_location/readme/CONSTRIBUTORS.rst]: Done

**Review by dreispt — APPROVED** [review]

**Pull request merged**